cmd/kubeadm/app/util/error.go

```diff
@@ -64,16 +66,35 @@ type preflightError interface {
 // checkErr formats a given error as a string and calls the passed handleErr
 // func with that string and an exit code.
 func checkErr(err error, handleErr func(string, int)) {
+
+	var msg string
+	if err != nil {
+		msg = err.Error()
```

Comment:
here we have the option to do:
```
msg = fmt.Sprintf("%s\nTo see the stack trace of this error execute with --v=5 or higher", err.Error())
```
so when someone logs a ticket they will already know how to ease our work by providing the stack trace.

however it might break users that are parsing a message from stderr.
(then again raw stderr messages should not be parsed)

Reply:
+1

Reply:
Sounds like a good addition.